pkgs/applications/audio/spotify/darwin.nix

```diff
@@ -0,0 +1,28 @@
+{ fetchurl, stdenv, lib, undmg }:
+
+stdenv.mkDerivation rec {
+  version = "1.0.42.151.g19de0aa6";
+  name = "spotify-mac";
+
+  src = fetchurl {
+    url = "http://download.spotify.com/Spotify.dmg";
```

Comment:
I assume there's no versioned url we can use, it would be nice if we could provide a source that will keep working.